internal/compiler/generator/rust.rs

```diff
@@ -1386,6 +1386,8 @@ fn generate_item_tree(
         )
     };
 
+    let embedding_function = if parent_ctx.is_some() { quote!(todo!()) } else { quote!(false) };
```

Comment:
maybe you want to add a text there, because if it panics in the generated code with just "todo" it will be really hard to find out where this todo comes from